app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -322,12 +335,132 @@ ListingService = ($http, $localStorage, $modal, $q, $state) ->
     # by default will just return the id, unless it finds a matching slug
     return if mapping[slug] then mapping[slug] else id
 
-  Service.listingIs = (listing, name) ->
-    Service.LISTING_MAP[listing.Id] == name
+  Service.listingIs = (name) ->
+    Service.LISTING_MAP[Service.listing.Id] == name
 
   Service.listingIsAny = (listing, names) ->
```

Comment:
I don't see this being used anywhere in the code.. can you clarify the function?